Polyfills/XMLHttpRequest/Source/XMLHttpRequest.cpp

```diff
@@ -149,6 +151,17 @@ namespace Babylon::Polyfills::Internal
         m_request.ResponseType(ResponseType::StringToEnum(value.As<Napi::String>().Utf8Value()));
     }
 
+    Napi::Value XMLHttpRequest::GetResponseHeader(const Napi::CallbackInfo& info)
+    {
+        const auto headerName{info[0].As<Napi::String>().Utf8Value()};
+        return Napi::Value::From(Env(), m_request.ResponseHeader(headerName));
+    }
+
+    Napi::Value XMLHttpRequest::GetAllResponseHeaders(const Napi::CallbackInfo&)
+    {
+        return Napi::Value::From(Env(), m_request.ResponseHeaders());
```

Comment:
Same for this:

[Return value](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/getAllResponseHeaders#return_value)

> A string representing all of the response's headers (except those whose field name is Set-Cookie) separated by [CRLF](https://developer.mozilla.org/en-US/docs/Glossary/CRLF), or null if no response has been received. If a network error happened, an empty string is returned.